### ISSUE TITLE
Messaging: User presence status

### ISSUE THREAD

**Issue by ruscoder** [voximplant/react-native-voximplant#59]
Hello! 

How to get users statuses at the moment when I subscribe to users statuses changes?

**Comment by Linoa65** [comment]
Hello !

Can you give more information about your problem ? Are you talking about ClientState ? Please give a snippet of your code to help you to answer to your question :)

Sincerely
Florient

**Comment by ruscoder** [comment]
@Linoa65 

I'm talking about UserStatus (messaging module in the `messaging` branch). We can subscribe to user status changing using `subscribe` method.

```
const messaging = Voximplant.getMessenger();
messaging.subscribe(['userid@my.app']);
```

and then we will receive `Voximplant.Messaging.MessengerEventTypes.SetStatus` event when the user `userid@my.app` changes his status using `setStatus` method.

```
messaging.setStatus(true);
```

The question is how to get the initial user status when I connect to the VI first time?

**Comment by Linoa65** [comment]
@ruscoder 

I'm looking inside Messenger class. It seems an event is raised when you subscribe to a messaging.

```js
_onSubscribe = (event) => {
        this._emit(MessengerEventTypes.Subscribe, event);
    };
```

I think first event raised on subscription is this one. I'm not sure, but some tests will help you.

**Comment by YuliaGrigorieva** [comment]
Hello!

Today we have released Voximplant React Native SDK 1.6.0 with Messaging API.

'messaging' branch will be completely removed from the repo on May 6th, 2019.

Best regards,
Yulia Grigorieva
